### DISCUSSION TITLE
Digital Ocean assets not showing on prod

### DISCUSSION THREAD

**Discussion by danklingjr** [statamic/cms#8663] [Troubleshooting] [answered]
I have assets building to DO on loc and I'm deploying to Netlify. Assets are publishing to the DO space fine but they're missing from prod and they're causing blog posts to break. Assuming this might be an env issue but I'm not sure where to start.

**Comment by duncanmcclean** [comment, marked as answer]
Have you added the relevant `.env` keys to your environment keys/secrets in Netlify? (Netlify will need to access DO spaces in order to read the assets)

**Comment by danklingjr** [comment]
Welp, duh. Env was in the gitignore and I never pushed the changes. I have not messed with the env file for Netlify until now so that makes sense.

**Reply by duncanmcclean** [reply]
You shouldn't include your `.env` file in your Git repo.

For Netlify, I believe there's a "Secrets" (or similar term) in the sites settings where you can add the relevant environment keys.

**Reply by danklingjr** [reply]
Found it listed under Environment Variables.
